draft-hodges-webauthn-registries.xml

```diff
@@ -42,164 +42,207 @@
         <email>Jeff.Hodges@PayPal.com</email>
       </address>
     </author>
-    
-    <date month="June" year="2016" />
-    
+    <author fullname="Giridhar Mandyam" initials="G.D."
+            surname="Mandyam">
+      <organization>Qualcomm Technologies Inc.</organization>
+      <address>
+        <postal>
+          <street>5775 Morehouse Drive</street>
+          <!-- Reorder these if your country does things differently -->
+          <city>San Diego</city>
+          <region>California</region>
+          <code>92121</code>
+          <country>USA</country>
+        </postal>
+        <phone>+1 858 651 7200</phone>
+        <email>mandyam@qti.qualcomm.com.com</email>
+        <!-- uri and facsimile elements may also be added -->
+      </address>
+    </author>
+
+    <date month="September" year="2016" />
+
     <keyword>webauthn</keyword>
     <keyword>attestation</keyword>
     <keyword>extensions</keyword>
     <keyword>registry</keyword>
-    
+
     <abstract>
-      <t>This specification defines IANA registries for W3C Web Authentication 
-      <xref target="WebAuthn"/>
-      attestation formats and extension identifiers.</t>
+      <t>
+        This specification defines IANA registries for W3C Web Authentication
+        <xref target="WebAuthn"/>
+        attestation formats and extension identifiers.
+      </t>
     </abstract>
-    
+
   </front>
-  
+
   <middle>
     <section title="Introduction">
       <t>
         This specification defines IANA registries for W3C Web Authentication <xref
-        target="WebAuthn"/> attestation formats and extension identifiers, and supplies initial 
+        target="WebAuthn"/> attestation formats and extension identifiers, and supplies initial
         entries within each registry.
       </t>
+      <t>
+        Candidate entries must be announced on the public Web Authentication mailing list,
+        public-webauthn@w3.org, so they can be discussed and evaluated for compliance 
+        to the requirements presented in this document before being added to its associated
+        registry.
+      </t>

```

Comment:
Apologies to @equalsJeffH.  After group discussion on Sep. 7 2016 call, it is clear that expert review involving posting to the public webauthn mailing list will be specified.  It just won't be specified in this document.  I will adjust the PR accordingly.